node-api/backend/backend.go

```diff
@@ -43,11 +44,12 @@ type Backend struct {
 	node types.ConsensusService
 
 	// the genesis data is cached here, written to once during initialization!
-	genesisHeader         atomic.Pointer[ctypes.BeaconBlockHeader]
-	genesisValidatorsRoot atomic.Pointer[common.Root]
-	genesisBlockRoot      atomic.Pointer[common.Root]
-	genesisTime           atomic.Pointer[math.U64]
-	genesisForkVersion    atomic.Pointer[common.Version]
+	genesisHeader      atomic.Pointer[ctypes.BeaconBlockHeader]
+	genesisBlockRoot   atomic.Pointer[common.Root]
+	genesisValidators  atomic.Pointer[[]*ctypes.Validator]
+	genesisTime        atomic.Pointer[math.U64]
+	genesisForkVersion atomic.Pointer[common.Version]
+	genesisState       atomic.Pointer[statedb.StateDB]
```

Comment:
all of this (except `genesisTime`) may be consolidated into just genesisState, since everything else can be derived from genesisTime or genesisState

Reply:
done, deriving others from genesisTime or genesisState